Detectors/EMCAL/simulation/src/Digitizer.cxx

```diff
@@ -104,6 +162,17 @@ void Digitizer::setEventTime(double t)
   mEventTime = t;
 }
 
+//_______________________________________________________________________
+void Digitizer::addNoiseDigits()
+{
+  for (int id = 0; id <= mGeometry->GetNCells(); id++) {
+    double energy = mRandomGenerator->Gaus(0, mSimParam->GetPinNoise());
+    double time = mRandomGenerator->Rndm() * mSimParam->GetTimeNoise();
```

Comment:
Would it be sufficient  to pre-generate a (CCDB) noise table and reuse the same numbers? There is potentially some speed to be gained.

Reply:
I don't think so. The noise is distributed randomly over the EMCAL surface. When storing in the CCDB we would have a static noise pattern.